src/backend_model.cc

```diff
@@ -103,62 +102,35 @@ TritonModel::Create(
   RETURN_IF_ERROR(BackendConfigurationSpecializeBackendName(
       backend_cmdline_config_map, backend_name, &specialized_backend_name));
 
-  std::string backend_libname;
-  RETURN_IF_ERROR(BackendConfigurationBackendLibraryName(
-      specialized_backend_name, &backend_libname));
-
-  // Get the path to the backend shared library. Search path is
-  // version directory, model directory, global backend directory.
-  const auto localized_model_path = localized_model_dir->Path();
-  const auto version_path =
-      JoinPath({localized_model_path, std::to_string(version)});
-  const std::string global_path =
-      JoinPath({backend_dir, specialized_backend_name});
-  std::vector<std::string> search_paths = {
-      version_path, localized_model_path, global_path};
-
-  std::string backend_libdir;
-  std::string backend_libpath;
-  std::string python_runtime_modeldir;
+  bool is_python_backend_based_backend = false;
```

Comment:
```suggestion
  bool is_python_based_backend = false;
```

Reply:
I was thinking about `python_binding_based_backend`, but simplicity is better. Updated: [Rename variable name](https://github.com/triton-inference-server/core/pull/285/commits/7da30c953bf5e058b978ff52a583fb061844fe03)